src/orion/algo/axoptimizer.py

```diff
@@ -0,0 +1,344 @@
+"""
+:mod:`orion.algo.axoptimizer` -- TODO
+========================================
+
+TODO: Write long description
+"""
+import contextlib
+import copy
+from typing import List, Optional
+
+try:
+    from ax.service.ax_client import AxClient
+    from ax.service.utils.instantiation import ObjectiveProperties
+
+    has_Ax = True
+except ImportError:
+    AxClient = None
+    ObjectiveProperties = None
+    has_Ax = False
+
+from orion.algo.base import BaseAlgorithm
+from orion.algo.space import Space
+from orion.core.utils import format_trials
+from orion.core.utils.flatten import flatten
+
+
+class AxOptimizer(BaseAlgorithm):
+    """TODO: Class docstring
```

Comment:
It would be good to add a link to https://ax.dev/ here and explain shortly that this is a wrapper using AxOptimizer that gives to multi-objective and optimization with constraints using Ax.